modules/ROOT/pages/index.adoc

```diff
@@ -2,29 +2,111 @@
 
-* *White-box testing:* This type of testing validates the behavior of individual APIs against the understanding of how their internals work. As part of this type of testing, you mock and simulate dependencies (for example, back-end systems and other APIs).
-* *Black-box testing:* This type of testing validates the overall behavior of an API and its real/live dependencies as a whole, based purely on inputs and outputs (i.e. without knowing or altering the API internals - no simulation or mocking).
-* *Runtime monitoring:* This type of monitoring ensures that deployed APIs are operating within expected performance in production environments. It makes use of behavioral test cases that use real inputs and expected outputs, and exercise dependencies, as in black-box testing.
+* _Public_ APIs are APIs deployed are accessible by anyone who is connected to the internet. For example, APIs that handle login requests for logging into banking websites are public APIs. Anyone with a browser and internet access can visit a login page and attempt to log in.
 
-A _monitor_ is a repeating schedule of one or more tests, referred to as a _suite_.
+* _Private_ APIs are APIs deployed in Anypoint Virtual Private Cloud in CloudHub and accessible only to people or applications that are in your private network. For example, when someone tries to log into a banking website, a request might be sent to an internal database to verify that person's credentials. The API for handling the request to the database would be considered a private API because it is within your private, secure network.
 
-You can get reports on the results of monitor executions. You can also set up integration with into a number of monitoring and issue-tracking systems.
+== Fundamentals of Monitors
+
+For each API or set of interdependent APIs that you want to monitor, you create a _monitor_, which is a named entity that consists of one test, one or more schedules that run the test at intervals, and information for connecting to any third-party applications that you set up to notify you of test failures.
+
+For example, you can create a monitor that runs a test twice daily to check whether an API that handles requests to a database is returning the correct data within the expected response time. The test can use the HTTP GET method to send a request to the endpoint at which the API is awaiting such requests. You can set two schedules: one to run the test at 9 in the morning, and another to run the test at 3 in the afternoon. You can also set the monitor to notify you via an email message, a Slack post, or another method through one of the supported third-party notification tools, when a test execution fails. You also get a summary of the test results whether or not a test execution fails, and can look up the results of past executions of the test.
+
+Scenarios that are more complicated are also possible. A test can run against multiple endpoints, making requests with any HTTP methods.
```

Comment:
The first sentence sounds odd to me, as if the scenario happens rather than you creating it. How about
```suggestion
You can also create more complicated tests. A test can run against multiple endpoints or make requests using any HTTP methods.
```

Reply:
Is there no text that lists all the following things that we are providing the fundamentals of?